### PR TITLE
ZeroPadding3D  shape bug

### PR DESCRIPTION
**Pull request by xingyu-long**
If the padding is 3 tuple of 2 ints: 
`((left_dim1_pad, right_dim1_pad), (left_dim2_pad, right_dim2_pad), (left_dim3_pad, right_dim3_pad))` When calculate the compute_output_shape, dim should add `padding[D][0] + padding[D][1] + input_shape[D]` instead of adding `2 * self.padding[D][0]`.
```
input = np.ones((2, 4, 5, 3, 2)) # channels_last
paddings = ((1, 2), (3, 4), (0, 2))
original compute_output_shape: return TensorShape([2, 8, 13, 7, 2])
Right size: TensorShape([2, 7, 12, 5, 2])
```
And `call` function holds the right shape.

### CODE REVIEW

**Review by fchollet — APPROVED** [review]
Thanks for the PR. Please add necessary unit tests to check & maintain this fix.

**Review by fchollet — CHANGES_REQUESTED** [review]
Thanks for the PR. Please add necessary unit tests to check & maintain this fix.

**Comment by xingyu-long** [comment]
> Thanks for the PR. Please add necessary unit tests to check & maintain this fix.

Already update and support in different configurations, ie. `channels_first`, `channels_last`.

File: tensorflow/python/keras/layers/convolutional_test.py [hunk 1]
```diff
@@ -730,35 +730,88 @@ def test_zero_padding_3d(self):
     input_len_dim2 = 5
     input_len_dim3 = 3
 
-    inputs = np.ones((num_samples, input_len_dim1, input_len_dim2,
-                      input_len_dim3, stack_size))
+    for data_format in ['channels_first', 'channels_last']:
```
Comment (fchollet) [inline tensorflow/python/keras/layers/convolutional_test.py:733]: Rather than using `for` loops, prefer using a parameterized test such as this one: https://github.com/tensorflow/tensorflow/blob/f760e5bb88054d7957bbaec2329f9175f75769a6/tensorflow/python/keras/layers/convolutional_test.py#L67

**Review by fchollet — CHANGES_REQUESTED** [review]
Thanks for the update. One comment; looks good otherwise!

File: tensorflow/python/keras/layers/convolutional_test.py (hunk 1 above)
Comment (xingyu-long) [reply tensorflow/python/keras/layers/convolutional_test.py]: > Rather than using `for` loops, prefer using a parameterized test such as this one:
> 
> https://github.com/tensorflow/tensorflow/blob/f760e5bb88054d7957bbaec2329f9175f75769a6/tensorflow/python/keras/layers/convolutional_test.py#L67

Already update and pass the tests.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!
